public/btn.primary-color.btn-sm.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="153" height="48" viewBox="0 0 153 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Using svg as a button is a bad practice, it will be hard to make it responsive. I advise you to create button component instead.